Dockerfile

```diff
@@ -0,0 +1,30 @@
+git submodule update --init --recursive
+yarn install --production
+
+# Build rapidsnark
+cd vendor/rapidsnark
+npm install
+git submodule init
+git submodule update
+npx task createFieldSources
+npx task buildProver
+
+# Build circuit
+cd ../build-artifacts/light_client_cpp
+make
+
+cd ../../../
+
+# Deploy the contract into Goerli
+cd beacon-light-client/solidity
+yarn hardhat deploy --network goerli
+
+# Start relayer
+redis-server
```

Comment:
You need to create a script that will launch the necessary servers and then specify this script as the `ENTRYPOINT` of the docker container.

Also, let's place the `Dockerfile` of the relayer in a sub-directory with the rest of the relayer code.

Reply:
The redis-server probably needs a config file where we will enable the persistent mode of redis. If the machine where the relayer is running is turned off and later relaunched (e.g. due to a power failure), we want the relayer to safely continue without losing any previous work.